src/config/config.ts

```diff
@@ -0,0 +1 @@
+ export const CONFIG = Object.freeze({"captions":{"hello":[{"content":"Hello! How are you?","start":0,"end":1536}],"bye":[{"content":"Bye! See you later!","start":0,"end":1512}]}});
```

Comment:
This file is always regenerated by the normal build process, right? Also, it's likely to be a very long, one-line file. We should probably .gitignore this file.

Reply:
Yes it does. I'll update .gitignore.